dev/NumberBox/InteractionTests/NumberBoxTests.cs

```diff
@@ -64,15 +64,24 @@ public void UpDownTest()
                 downButton.InvokeAndWait();
                 Verify.AreEqual(0, numBox.Value);
 
-                Log.Comment("Change Step value to 5");
-                RangeValueSpinner stepNumBox = FindElement.ByName<RangeValueSpinner>("StepNumberBox");
-                stepNumBox.SetValue(5);
+                Log.Comment("Change SmallChange value to 5");
+                RangeValueSpinner smallChangeNumBox = FindElement.ByName<RangeValueSpinner>("SmallChangeNumberBox");
+                smallChangeNumBox.SetValue(5);
                 Wait.ForIdle();
 
                 Log.Comment("Verify that up button increases value by 5");
                 upButton.InvokeAndWait();
                 Verify.AreEqual(5, numBox.Value);
 
+                Log.Comment("Change LargeChange value to 50");
+                RangeValueSpinner largeChangeNumBox = FindElement.ByName<RangeValueSpinner>("LargeChangeNumberBox");
+                largeChangeNumBox.SetValue(50);
+                Wait.ForIdle();
+
+                Log.Comment("Verify that up button increases value by 50");
+                upButton.InvokeAndWait();
```

Comment:
The up button always does small increase doesn't it?

Reply:
Yes, the comment was wrong. i fixed the test so it sends PageUp to test the large change behavior.